docs/execute-methods.md

```diff
@@ -108,7 +108,7 @@ strategy | string | no | One of possible app installation strategies on real dev
 ### mobile: isAppInstalled
 
 Checks whether the given application is installed on the device under test.
-Offload app could be handled as not installed.
+An offload application could be handled as not installed.
```

Comment:
maybe it makes sence to make it to a link. Not everybody knows what `offload application` means

Reply:
Yes, I'll add link later